### ISSUE TITLE
Strange literal in the code

### ISSUE THREAD

**Issue by kiskit** [Fiware/context.Orion-LD#1297]
Hi guys, 

I'm not sure this has any effect, but I guess it probably does. I haven't tested it to be completely honest. That said, in qVariableFix.cpp, there is a strange value at line 275:

```C
      if ((strcmp(mdNameP, "observedAt") == 0) || (strcmp(mdNameP, "modifiededAt") == 0) || (strcmp(mdNameP, "createdAt") == 0))
```
I'll go as far as supposing you meant ''modifiedAt'' instead of ''modifiededAt''.

**Comment by kzangeli** [comment]
Thanks Hawkeye!
Yeah, of course, that's a bug, with uppercase 'B' !
I'll fix it asap

**Comment by kzangeli** [comment]
Fixed in #1299
